payments-core/src/test/java/com/stripe/android/view/ShippingInfoWidgetTest.kt

```diff
@@ -182,6 +173,9 @@ class ShippingInfoWidgetTest {
         cityEditText.setText("Fake City")
         postalEditText.setText("12345")
         stateEditText.setText("CA")
+        phoneEditText.setText("8675555309")
```

Comment:
This field must not have been required for PaymentFlowActivity, so it wasn't set in this test. But now that we are just using a generic ShippingInfoWidget, it needs to be set